locations/spiders/kfc_cz.py

```diff
@@ -0,0 +1,17 @@
+from unidecode import unidecode
+
+from locations.categories import Categories
+from locations.spiders.kfc import KFC_SHARED_ATTRIBUTES
+from locations.storefinders.amrest_eu import AmrestEUSpider
+
+class KFCCZSpider(AmrestEUSpider):
```

Comment:
Is it worth splitting these off to a different PR? I can see from the issues raised you are sort of headed in this direction

Reply:
Yes, and I apologise for including here without explanation. I went to implement store finder detection for `AmrestEUSpider` and realised this storefinder was a bit of a mess and needed to be fixed prior to implementing store finder detection. My intent with rebasing this branch once it is feature complete is to structure a number of stacked branches in this order:

1. Basic store finder detection and automatic spider generation framework (structured_data.py, automatic_spider_generator.py and sf.py)
2. Implementation of store finder detection for each store finder (one branch per store finder). The rewrite of AmrestEUSpider to support store finder detection would occur in it's own branch including updates to existing spiders that use AmrestEUSpider. The intent is not to rewrite stuff unless it's necessary to enable store finder detection.

Reply:
Eh, when inspiration takes hold, lean in to it and force push to hide the mess :P